crates/sui-types/src/gas_model/gas_v2.rs

```diff
@@ -269,9 +278,47 @@ mod checked {
                 0,
                 None,
                 SuiCostTable::unmetered(),
+                None,
+            )
+        }
+
+        fn get_gas_vals(&self) -> String {
+            let gas_used = self.gas_status.gas_used_pre_gas_price();
+            let gas_rounded = if gas_used > 0 && gas_used % 1000 == 0 {
+                gas_used * self.gas_price
+            } else {
+                ((gas_used / 1000) + 1) * 1000 * self.gas_price
+            };
+            let computation_cost = if self.gas_budget <= gas_rounded {
+                self.gas_budget
+            } else {
+                gas_rounded
+            };
+
+            let (instr_count, stack_height, stack_size) = self.gas_status.get_status_info();
+
+            format!(
+                "{}, {}, {}, {}, {}",
```

Comment:
I do not know how I feel writing out only values, it makes everything so obscure...